src/meta/src/hummock/hummock_manager.rs

```diff
@@ -864,6 +869,14 @@ where
             self.check_state_consistency().await;
         }
 
+        self.env
+            .notification_manager()
+            .notify_frontend(
```

Comment:
Seems notify_frontend can be blocking due to `NotificationManagerCore::notify_frontend`?
`HummockManager::commit_epoch` shouldn't be blocked by it.